### PR TITLE
Feature/web components

### PR DESCRIPTION
**Pull request by CaliZam**

### CODE REVIEW

**Comment by alex-martinez-jativa** [comment]
Dani, you should to implement Hooks in some components. is ok to use classes in addition to hooks

**Comment by CaliZam** [comment]
> Dani, you should to implement Hooks in some components. is ok to use classes in addition to hooks

Hi Alex, I know, I tried, but i ran out of time, once I have some time i will try to implement in all compos. Thnks

**Comment by CaliZam** [comment]
> Dani, you should to implement Hooks in some components. is ok to use classes in addition to hooks

Hi Alex, I already use Hooks and classes, could you check again. Thanks

File: staff/daniel-zamora/web-components/my-app/logic/authenticateUser.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function authenticateUser(email, password) {
```
Comment (alexiparkhio) [inline staff/daniel-zamora/web-components/my-app/logic/authenticateUser.js:1]: no camelCase on file names unless they're components. Please use `authenticate-user.js` . This is relatable to other logic files you have here as well

Comment (CaliZam) [reply staff/daniel-zamora/web-components/my-app/logic/authenticateUser.js]: Hi Alex, 

I just figure out was in my-App, not my app react, changed already. Thanks

**Pull request merged**